src/interfaces.rs

```diff
@@ -168,34 +168,39 @@ impl Interfaces {
                     Interfaces::validate_float(obj.1)?;
 
                     let mapping = self
-                        .get_mapping(interface_name, &format!("{}{}", interface_path, obj.0))
-                        .ok_or_else(|| AstarteError::SendError("Mapping doesn't exist".into()))?;
+                        .get_mapping(
+                            interface_name,
+                            &format!("{}/{}", interface_path.trim_end_matches('/'), obj.0),
```

Comment:
Why trimming a `/` and re-introducing it in the format string?

Reply:
so it works regardless if it was there or not

Reply:
Can you tell me what this difference is

Reply:
Check this example (Elixir syntax):
```
iex(6)> Astarte.Device.send_datastream(pid, "org.astarte-platform.genericsensors.Geolocation", "/test/", %{"accuracy" => 42.42})
{:error, :invalid_path}
iex(7)> Astarte.Device.send_datastream(pid, "org.astarte-platform.genericsensors.Geolocation", "/test", %{"accuracy" => 42.42}) 
:ok
```